train.py

```diff
@@ -53,13 +53,18 @@ def train(
         default=-1
     ),
     logging_steps: int = Input(
-        description="number of steps between logging epoch & loss", default=1
+        description="number of steps between logging epoch & loss", default=100
```

Comment:
curious what the latency impact would be if we lowered this - in general I think it's a great idea to have a high # of steps here, but since we have no other way of surfacing loss curves to users atm, perhaps we should keep this lower?

Reply:
Lowered!